src/Commands/Env/DeployCommand.php

```diff
@@ -9,11 +9,20 @@ class DeployCommand extends TerminusCommand
 {
 
     /**
+     * Contructor that supports injection for unit testing.
+     */
+    public function __construct($sites = null)
+    {
+        parent::__construct();
+        $this->sites = $sites ? $sites : new Sites();
```

Comment:
I would get behind bumping up a DI strategy sooner rather than later. Would really simplify testing and not make mine smell so bad.